static/js/tab_bar.js

```diff
@@ -8,19 +8,22 @@ function get_sub_count(current_stream) {
 
 function get_formatted_sub_count(current_stream) {
     let sub_count = get_sub_count(current_stream);
+    let suffix = '';
     if (sub_count >= 1000) {
         // parseInt() is used to floor the value of division to an integer
-        sub_count = parseInt(sub_count / 1000, 10) + "k";
+        sub_count = parseInt(sub_count / 1000, 10);
+        suffix = 'k';
     }
+    sub_count = i18n.t('__count____suffix__', {count: parseInt(sub_count / 1000, 10), suffix: suffix});
```

Comment:
Not sure if this is the right approach, but the numbers need to go through i18n in both `>=1000` and `<1000` cases